docs/ext/generate_release_notes.py

```diff
@@ -0,0 +1,41 @@
+"""
+Sphinx extension to generate ``releases/releases_inc.rst`` file
+"""
+# TODO: For now we manually generate the content until we have the
+#  tmt:autostory to more natively integrate it.
+
+import typing
+
+if typing.TYPE_CHECKING:
+    from sphinx.application import Sphinx
+
+
+import tmt
+from tmt._compat.pathlib import Path
+
+
+def generate_release_notes(app: "Sphinx") -> None:
+    """
+    Generate ``releases/releases_inc.rst`` file
+    """
+
+    # We will need a logger...
+    logger = tmt.Logger.create()
+    logger.add_console_handler()
+    tree = tmt.Tree(logger=logger, path=Path(app.confdir / "releases"))
+
+    release_inc = app.confdir / "releases/release.rst.inc"
+    with release_inc.open("w") as doc:
+        # The release notes structure is /<release>/<issue|note>
+        for release in tree.stories(names=[r"^/[\d\.]+$"], whole=True):
```

Comment:
We should sort the releases in the descending order to get the latest release always at the top.

Reply:
Oh, right. Hmm, alphabetical order would be an issue with variable number of digits, but I think we can just feed it through `packaging.Version` and let it automagically sort it.